data/config-schema/src/main/resources/net/corda/schema/configuration/cryptoLibrary/1.0/corda.cryptoLibrary.json

```diff
@@ -306,16 +301,8 @@
                             "type": "object",
                             "properties": {
                               "configSecret": {
-                                "type": "object",
-                                "properties": {
-                                  "encryptedSecret": {
-                                    "type": "string"
-                                  }
-                                },
-                                "additionalProperties": false,
-                                "required": [
-                                  "encryptedSecret"
-                                ]
+                                "type": ["object", "string"],
+                                "additionalProperties": true
```

Comment:
I don't think this is right.
I think `passphrase` needs to be of type `string`. 
Turning a string into a secret is optional and we don't expose that in the schema.

Reply:
It was changed to string now as the https://github.com/corda/corda-runtime-os/pull/2071 was merged